packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 2.3.0
 
+* Adds options for gesture handling and tilt controls on web.
```

Comment:
Looks like you accidentally merged into another version update on the last update.

Reply:
Although it looks like the pubspec version change is missing?

Reply:
Okay I think I fixed it